inbm/dispatcher-agent/dispatcher/sota/os_factory.py

```diff
@@ -99,12 +99,14 @@ def create_os_updater(self) -> OsUpdater:
         pass
 
     @abstractmethod
-    def create_snapshotter(self, sota_cmd: str, snap_num: Optional[str], proceed_without_rollback: bool) -> Snapshot:
+    def create_snapshotter(self, sota_cmd: str, snap_num: Optional[str], 
```

Comment:
Ensure that the `device_reboot` parameter is passed to all `create_snapshotter` calls in the `os_factory.py` file. [important]